pyairtable/orm/model.py

```diff
@@ -170,6 +172,20 @@ def _validate_class(cls) -> None:
                 )
             )
 
+    @classmethod
+    def _get_meta_request_kwargs(cls):
+        # Called by modify_kwargs to modify kwargs passed to `all()` & `first()`.
+        return {
+            "user_locale": None,
+            "cell_format": "json",
+            "time_zone": None,
+            "return_fields_by_field_id": (
+                cls._get_meta("use_field_ids")
+                if cls._get_meta("use_field_ids")
+                else False
```

Comment:
I think this can be simplified to `cls._get_meta("use_field_ids", default=False)`